ipatests/test_integration/test_crlgen_manage.py

```diff
@@ -302,3 +302,8 @@ def test_uninstall_last_master_does_not_require_ignore_last_of_role(self):
         check_crlgen_enable(self.replicas[0])
         # call uninstall without --ignore-last-of-role, should be OK
         self.master.run_command(['ipa-server-install', '--uninstall', '-U'])
+
```

Comment:
`test_uninstall_last_master_does_not_require_ignore_last_of_role` is a part of testsuite, so we expect that once it can fail (i.e. if uninstallation without `--ignore-last-of-role` does not work). 
In this case proper uninstallation will be needed